.circleci/config.yml

```diff
@@ -425,6 +425,27 @@ jobs:
           name: Update RPM repository
           command: './scripts/build/update_repo/update-rpm.sh "oss" "$GPG_KEY_PASSWORD" "$CIRCLE_TAG" "dist"'
 
+  build-oss-msi:
+    docker:
+      - image: grafana/grafana-oss-ci-msi-build
```

Comment:
Lock down the image to a version before merging. Using latest is problematic and make it hard to build old versions of the code when we need to change things in both the image and the the grafana repo.